package.json

```diff
@@ -7,10 +7,10 @@
     "test": "npx codeceptjs run --steps",
     "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @smoke-tests --reporter mocha-multi --verbose",
     "test:ga-smoke": "MOCHAWESOME_REPORTFILENAME=ga-smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @ga-smoke-tests --reporter mocha-multi --verbose",
-    "test:api": "MOCHAWESOME_REPORTFILENAME=api npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
+    "test:api": "npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
     "test:e2e": "MOCHAWESOME_REPORTFILENAME=e2e npx codeceptjs run --grep @e2e-tests --reporter mocha-multi --verbose",
     "test:e2e-multiparty": "MOCHAWESOME_REPORTFILENAME=e2e-multiparty npx codeceptjs run --grep @multiparty-e2e-tests --reporter mocha-multi --verbose",
-    "test:functional": "MOCHAWESOME_REPORTFILENAME=functional npx codeceptjs run --grep '@e2e-tests|@multiparty-e2e-tests' --reporter mocha-multi --verbose",
```

Comment:
Are we not supposed to revert the changes to the package.json after testing in local?

Reply:
Removed local Changes